tests/components/images.test.ts

```diff
@@ -0,0 +1,178 @@
+  model = new Model();
+  ({ app, parent } = await mountSpreadsheet(fixture, { model, fileUploader }));
+  model = parent.model;
+  await nextTick();
+});
+
+afterEach(() => {
+  app.destroy();
+  fixture.remove();
+});
+
+describe("image component tests", function () {
+  test("insert image", async () => {
+    const sheetId = model.getters.getActiveSheetId();
+    const originalLenght = model.getters.getImageIds(sheetId).length;
+    await simulateClick(".o-topbar-menu[data-id='insert']");
+    await simulateClick(".o-menu-item[data-name='insert_image']");
+    const newLenght = model.getters.getImageIds(sheetId).length;
+    expect(newLenght).toEqual(originalLenght + 1);
+  });
+
+  test("Click on Delete button will delete the image", async () => {
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: "Sheet1" } });
+    await nextTick();
+    expect(fixture.querySelector(".o-figure")).not.toBeNull();
+    await simulateClick(".o-figure");
+    expect(document.activeElement).toBe(fixture.querySelector(".o-figure"));
+    expect(fixture.querySelector(".o-image-menu-item")).not.toBeNull();
+    await simulateClick(".o-image-menu-item");
+    expect(fixture.querySelector(".o-menu")).not.toBeNull();
+    await simulateClick(".o-menu div[data-name='delete']");
+    expect(() => model.getters.getImage(imageId)).toThrow();
+  });
+
+  test("Can copy/paste an image with its context menu", async () => {
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    await nextTick();
+    const image = model.getters.getImage(imageId);
+    await simulateClick(".o-figure");
+    await simulateClick(".o-image-menu-item");
+    await simulateClick(".o-menu div[data-name='copy']");
+    paste(model, "A1");
+    expect(model.getters.getImageIds(sheetId).length).toEqual(2);
+    const imageIds = model.getters.getImageIds(sheetId);
+    expect(model.getters.getImage(imageIds[1])).toEqual(image);
+    expect(model.getters.getImage(imageIds[0])).toEqual(image);
+  });
+
+  test("Can cut/paste an image with its context menu", async () => {
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    await nextTick();
+    const image = model.getters.getImage(imageId);
+    const originalLenght = model.getters.getImageIds(sheetId).length;
+    await simulateClick(".o-figure");
+    await simulateClick(".o-image-menu-item");
+    await simulateClick(".o-menu div[data-name='cut']");
+    paste(model, "A1");
+    const newLenght = model.getters.getImageIds(sheetId).length;
+    expect(newLenght).toEqual(originalLenght);
+    const imageIds = model.getters.getImageIds(sheetId);
+    expect(model.getters.getImage(imageIds[0])).toEqual(image);
+  });
+
+  test("can export a image", async () => {
```

Comment:
This test should be in teh plugin test file, not the component test file. We should also have a test for the import too.